Feature BJS2-16690 done

maksimovsky opened the pull request.

2V57 committed:
BJS2-16720
Resolved
Configured Swagger and added annotations (Swagger).

maksimovsky committed:
Feature BJS2-16690 done

MaxVodolazhsky commented on src/main/java/school/faang/user_service/service/SkillService.java:
```diff
@@ -0,0 +1,92 @@
+    public List<SkillCandidateDto> getOfferedSkills(long usedId) {
+        List<Skill> offeredSkills = skillRepository.findSkillsOfferedToUser(usedId);
+        List<SkillDto> offeredSkillDtos = skillMapper.toDto(offeredSkills);
+        Map<SkillDto, Long> map = new HashMap<>();
```
minor
Данное название переменной (`map`), имеет очень узкую смысловую нагрузку. Возможно `skills` -  подойдет.

MaxVodolazhsky reviewed (commented):
В остальном все ок, давай поправим нейминг чутка.

maksimovsky committed:
Изменено название переменной в сервисе

Добавлены импорты в тесты, так как названия пакетов стали другими (service -> Service, controller -> Controller)

maksimovsky committed:
Изменено название переменной в сервисе

Добавлены импорты в тесты, так как названия пакетов стали другими (service -> Service, controller -> Controller)

maksimovsky committed:
Revert "BJS2-16720"

This reverts commit c6f635fcb16795527f916a0c60b3de70cbaf22a3.

maksimovsky committed:
Revert "Изменено название переменной в сервисе"

This reverts commit df0046edfdbd1b207733237f7759efc3f01a7cec.

maksimovsky committed:
Revert "Изменено название переменной в сервисе"

This reverts commit 83ff8e885ee18db5e6e564eed6e7eee83991b59c.

maksimovsky committed:
Changed variable name in service
